diff -ur Pantomime-orig/Source/IMAPStore.m Pantomime/Source/IMAPStore.m
--- Pantomime-orig/Source/IMAPStore.m	2003-07-24 15:30:36.000000000 -0600
+++ Pantomime/Source/IMAPStore.m	2003-08-20 15:34:02.000000000 -0600
@@ -257,16 +257,16 @@
 			 password: (NSString*) thePassword
 {
   NSString *aPassword;
-  NSRange aRange;
-
+  NSRange aRange, aRange2;
   // We first retain the username for future use
   username = theUsername;
   RETAIN(username);
 
   // We must verify if we must quote the password
   aRange = [thePassword rangeOfCharacterFromSet: [NSCharacterSet punctuationCharacterSet]];
+  aRange2 = [thePassword rangeOfCharacterFromSet: [NSCharacterSet whitespaceCharacterSet]];
   
-  if ( aRange.length )
+  if ( aRange.length || aRange2.length )
     {
       aPassword = [NSString stringWithFormat: @"\"%@\"", thePassword];
     }
Only in Pantomime/Source: IMAPStore.m~
